.github/workflows/build.yaml

```diff
@@ -28,3 +30,11 @@ jobs:
       build_type: branch
       script: "ci/build_wheel.sh"
       matrix_filter: ${{ needs.compute-matrix.outputs.BUILD_MATRIX }}
+  publish-wheels:
+    needs:
+      - build-wheels
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/wheels-publish.yaml@branch-24.02
+    with:
+      build_type: pull-request
```

Comment:
The changes we made to get things working in pr.yaml don't apply in build.yaml. You want to undo the changes to this file in https://github.com/rapidsai/pynvjitlink/pull/25/commits/3e1fcf2f27f8eafda1fc8981968acbc6e72b4681